cloud/connecting-algorithm/src/constants.py

```diff
@@ -0,0 +1,11 @@
+# This file contains constants that will be used in lambda_handler()
+
+# How many weeks we will take into account regarding repeat matches
+MATCH_COOLDOWN = 5
+
+# The userId of the 'Joker' user
+JOKER_USER_ID = '17d2f33d-67e0-40ab-977e-73d1580e990d'
```

Comment:
```suggestion
JOKER_USER_ID = '81f3816a-edb5-4352-9f9a-953f23d2f6b1'
```

I've already got my user ID so lets just set that up now